### DISCUSSION TITLE
Subdivide Until All Cell Areas Under Threshold

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#2632] [Q&A] [answered]
Is there a way to subdivide such that at the end of the process all cells have an area less than or equal to a provided threshold?

**Comment by akaszynski** [comment, marked as answer]
[subdivide_adaptive](https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyData.subdivide_adaptive.html?highlight=subdivide#subdivide-adaptive) should be what you're looking for.
